docs/pages/versions/v44.0.0/sdk/filesystem.md

```diff
@@ -796,19 +796,19 @@ Alias to [FileSystem.copyAsync(options)](#filesystemcopyasyncoptions)
+| `getInfoAsync`            | `file:///`,<br/>`content://`,<br/>`asset://`,<br/>no scheme**\***                                                                          | `file://`,<br/>`ph://`,<br/>`assets-library://`                                                 |
+| `readAsStringAsync`       | `file:///`,<br/>`asset://`,<br/>[SAF URI](#saf-uri)                                                                                        | `file://`                                                                                       |
+| `writeAsStringAsync`      | `file:///`,<br/>[SAF URI](#saf-uri)                                                                                                        | `file://`                                                                                       |
+| `deleteAsync`             | `file:///`,<br/>[SAF URI](#saf-uri)                                                                                                        | `file://`                                                                                       |
+| `moveAsync`               | Source:<br/>`file:///`,<br/>[SAF URI](#saf-uri)<br/><br/>Destination:<br/>`file://`                                                        | Source:<br/>`file://`<br/><br/>Destination:<br/>`file://`                                       |
+| `copyAsync`               | Source:<br/>`file:///`,<br/>`content://`,<br/>`asset://`,<br/>[SAF URI](#saf-uri),<br/>no scheme**\***<br/><br/>Destination:<br/>`file://` | Source:<br/>`file://`,<br/>`ph://`,<br/>`assets-library://`<br/><br/>Destination:<br/>`file://` |
+| `makeDirectoryAsync`      | `file:///`                                                                                                                                 | `file://`                                                                                       |
+| `readDirectoryAsync`      | `file:///`                                                                                                                                 | `file://`                                                                                       |
+| `downloadAsync`           | Source:<br/>`http://`,<br/>`https://`<br/><br/>Destination:<br/>`file:///`                                                                 | Source:<br/>`http://`,<br/>`https://`<br/><br/>Destination:<br/>`file://`                       |
+| `uploadAsync`             | Source:<br/>`file:///`<br/><br/>Destination:<br/>`http://`<br/>`https://`                                                                  | Source:<br/>`file://`<br/><br/>Destination:<br/>`http://`<br/>`https://`                        |
+| `createDownloadResumable` | Source:<br/>`http://`,<br/>`https://`<br/><br/>Destination:<br/>`file:///`                                                                 | Source:<br/>`http://`,<br/>`https://`<br/><br/>Destination:<br/>`file://`                       |     |
```

Comment:
Could you update https://github.com/expo/expo/blob/main/docs/pages/versions/unversioned/sdk/filesystem.md instead? The `unversioned` is used to generate the docs for future releases 😉 We won't release this change to existing SDK44, so we also shouldn't update its docs.